sdk/eventhub/event-hubs/test/stress/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# internal users should provide MCR registry to build via 'docker build . --build-arg REGISTRY="mcr.microsoft.com/mirror/docker/library/"'
+# public OSS users should simply leave this argument blank or ignore its presence entirely
+ARG REGISTRY=""
+FROM ${REGISTRY}node:18
+RUN mkdir -p /app/src/
+ADD ./app/src/*.ts /app/src/
+ADD ./app/package.json /app/package.json
+ADD ./app/tsconfig.json /app/tsconfig.json
+WORKDIR /app
+RUN npm install && npm run build
```

Comment:
nit: I feel that this should be a multiple stage build: one stage to build the ts files then another stage is to deploy the built and maybe bundled js files and dependencies to another container to run. no actions required. just my 2 cents